bindata/network/ovn-kubernetes/self-hosted/004-config.yaml

```diff
@@ -48,6 +48,12 @@ data:
 {{- if .OVN_MULTI_NETWORK_ENABLE }}
     enable-multi-network=true
 {{- end }}
+{{- if .OVN_NETWORK_SEGMENTATION_ENABLE }}
```

Comment:
When OVN_NETWORK_SEGMENTATION_ENABLE becomes true control-plane won't restart, is this expected?

Reply:
do you mean if someone disables the feature? why so?

Reply:
pat explained offline this is for users going from GA to TP (can't think why someone would support this as a day2 thing but here we are); so this needs to also be done on control plane config-maps and I have fixed this.